README.md

```diff
@@ -41,7 +47,7 @@ A: Likely not. Rendering the entirety of the maps for DS3, Bloodborne, and Sekir
 ## Credits:
 * Katalash - primary author
 * philiquaz - major contributor to integrated param editor
-* george - contributor to integrated text editor and other features
+* george - stable boy
```

Comment:
lgtm

Reply:
lol oops. didn't mean to include that, but frankly I do'nt mind it as my official title